.github/CODEOWNERS

```diff
@@ -18,11 +18,12 @@
 # all Packer resources and clients are owned by the cloud-packer team
 *packer* @hashicorp/cloud-packer
 
-# all Vault resources and clients are owned by the vault-cloud team
-*vault* @hashicorp/vault-cloud
-
 # all Vault Secrets resources and clients are owned by cloud-vault-secrets team
 *vaultsecrets* @hashicorp/cloud-vault-secrets
+*vault_secrets* @hashicorp/cloud-vault-secrets
```

Comment:
I am guessing this should work, right?

Reply:
Theory Will Take You Only So Far. Let's try it!